federatedml/transfer_variable/definition/transfer_conf.json

```diff
@@ -1188,108 +1327,5 @@
     "guest"
    ]
   }
- },
- "HeteroGMFTransferVariable": {
```

Comment:
This line delete the transfer variable definition of GMF，is this ok？

Reply:
the transfer_variable folder is under federatedrec, so this part can ignor